wire/core/WireMail.php

```diff
@@ -328,11 +328,12 @@ public function param($value) {
 	 * @return this 
 	 *
 	 */
-	public function attachment($value) {
+	public function attachment($value, $filename = '') {
 		if(is_null($value)) {
 			$this->mail['attachments'] = array();
 		} else if(is_file($value)) { 
-			$this->mail['attachments'][] = $value; 
+			$filename = $filename ?: basename($value);
+			$this->mail['attachments'][$filename] = $value; 
```

Comment:
Wouldn't it be better to swap $filename and $value, to avoid override a previously added file?
->attachment($value1, ''imag.jpg")
->attachment($value2, ''imag.jpg")